lib/sift/filter.rb

```diff
@@ -73,12 +70,44 @@ def valid_scope_params?(scope_params)
       scope_params.is_a?(Array) && scope_params.all? { |symbol| symbol.is_a?(Symbol) }
     end
 
-    def handler
-      parameter.handler
```

Comment:
We can't have Parameter be responsible for the handler, because scopes need more than just the "chief" parameter to handle.